### PR TITLE
Laravel 5.4 Illuminate/Routing/Route class changes.

### PR DESCRIPTION
**Pull request by denpamusic**
getUri() method was renamed to uri(), and getMethods() method was
renamed to methods().
See https://laravel.com/api/5.4/Illuminate/Routing/Route.html

### CODE REVIEW

**Comment by denpamusic** [comment]
Sorry, I'll add method_exist check.

**Comment by andrewmclagan** [comment]
any chance of a merge on this anytime soon?

File: src/Routing/Adapter/Laravel.php [hunk 1]
```diff
@@ -5,8 +5,8 @@
 use Illuminate\Http\Request;
 use Illuminate\Routing\Route;
 use Illuminate\Routing\Router;
-use Illuminate\Routing\RouteCollection;
 use Dingo\Api\Contract\Routing\Adapter;
+use Illuminate\Routing\RouteCollection;
```
Comment (bkintanar) [inline src/Routing/Adapter/Laravel.php]: Is this change necessary? I think we can revert this.

Comment (denpamusic) [inline src/Routing/Adapter/Laravel.php]: No, it's not. It actually make no sense for me, but Style CI complains about it https://styleci.io/analyses/qoQRrp

Comment (andrewmclagan) [inline src/Routing/Adapter/Laravel.php]: Agree, should follow the code style? ;-)

Comment (bkintanar) [inline src/Routing/Adapter/Laravel.php]: Well that's weird.. @GrahamCampbell any input on this weird Style CI analysis?

Comment (andrewmclagan) [inline src/Routing/Adapter/Laravel.php]: Any chance to merge this PR with the suggested style-ci changes?

**Comment by jasonlewis** [comment]
5.4 has been tagged stable for less than 24 hours. In no way should you expect this package to work out of the box with it.

File: src/Routing/Adapter/Laravel.php (hunk 1 above)
Comment (crynobone) [inline src/Routing/Adapter/Laravel.php]: > Is this change necessary? I think we can revert this.

Same length, it would then order the same length by alphanumeric order, so `D` before `I`.

**Comment by denpamusic** [comment]
Of course not. I mostly shared this PR to save time for developers like myself who tried updating dev environment to see what breaks with 5.4.

**Comment by jasonlewis** [comment]
Jolly good!
